pyEpiabm/pyEpiabm/tests/test_unit/test_sweep/test_intervention_sweep.py

```diff
@@ -0,0 +1,40 @@
+import unittest
+
+import pyEpiabm as pe
+from pyEpiabm.sweep import InterventionSweep
+from pyEpiabm.property import InfectionStatus
+from pyEpiabm.tests.test_unit.parameter_config_tests import TestPyEpiabm
+
+
+class TestInterventionSweep(TestPyEpiabm):
+    """Test the 'InterventionSweep' class.
+    """
+
+    @classmethod
+    def setUpClass(cls) -> None:
+        super(TestInterventionSweep, cls).setUpClass()
+        cls.interventionsweep = InterventionSweep()
+
+        # Construct a population with 8 persons
+        cls._population = pe.Population()
+        cls._population.add_cells(1)
+        cls._population.cells[0].add_microcells(1)
+        cls._population.cells[0].microcells[0].add_people(8)
+        for i in range(8):
+            person = cls._population.cells[0].microcells[0].persons[i]
+            person.update_status(InfectionStatus(i + 1))
+        cls.interventionsweep.bind_population(cls._population)
+
+    def test_bind_population(self):
+        self.assertEqual(len(self.interventionsweep.
+                             intervention_params['vaccine_params']), 9)
```

Comment:
Slightly confused by this - is this just checking that we can read in all the correct parameters (and no more)?

Reply:
Ahh makes sense - well no harm in additional tests but just left a little hardcoded! So long as we still expect 9 parameters (presumably for vaccinations only) even when other invention classes are added, then this is fine!